apps/kahoot-it-clone/src/app/quiz/quiz.component.css

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: grid;
+  grid-template-columns: 20% 80%;
+  height: 100%;
+}
+
+.quizBankComponent {
+  background-color: lightpink;
+}
+
+.questionComponent {
+  background-color: violet;
+}
```

Comment:
Definitely need to look into better colors. Normally the default colors are bad, you could look up some interesting color pallets online and use them.